### ISSUE TITLE
[A11Y] - All Editorial View | Announcement workflow, some input fields are missing labels

### ISSUE THREAD

**Issue by Tribunal33** [pkp/pkp-lib#12707]
## Issue Description
Concern was brought up about the Announcements Page tab.  It would appear that some elements are not properly labelled.

Here was the note from the auditor :
> ...When I tried to edit the description, I’m not sure which text field I was editing. They don’t seem to be labelled. This was the same on both.

After some investigation the Title will say the correct input label but some other ones like Short Description won't say anything.  This looks like it is part of tiny MCE which only describes the text box but not the label.

<img width="790" height="124" alt="Image" src="https://github.com/user-attachments/assets/27c661d3-6dc4-4776-8f3a-cfabf2af7a92" />
 

## How to reproduce / Current behavior
 Make sure announcements is enabled > Website Settings > Announcements > Enable announcements
Navigate to editorial backend with accesss to Announcements.
Select Announcements > Edit existing or Add Announcement
Tab through all the fields with a screen reader.  

## What application are you using?
 OJS, OMP or OPS version X.X.X-X

## Stack used 
- Device: Laptop
- OS: windows 11
- Browser: Chrome
- Version: Latest

## Testing method
Manual with screenreader

## Action
Fix the inputs that are marked with an ❌ 
- Title ✅ 
- Short Description ❌ 
- Announcement ❌ 
- Image ❌ 
- Expiry Date ✅ 
- Send File ✅ 

## Additional information
// Please add any screenshots, code snippets, or other information we can use to investigate this issue.
